sound/soc/sof/ipc4-topology.h

```diff
@@ -15,7 +15,18 @@
 #define SOF_IPC4_FW_PAGE(x) ((((x) + BIT(12) - 1) & ~(BIT(12) - 1)) >> 12)
 #define SOF_IPC4_FW_ROUNDUP(x) (((x) + BIT(6) - 1) & (~(BIT(6) - 1)))
 
-#define SOF_IPC4_MODULE_LL      BIT(5)
+#define SOF_IPC4_MODULE_LOAD_TYPE		GENMASK(3, 0)
+#define SOF_IPC4_MODULE_AUTO_START		BIT(4)
```

Comment:
where are these 2 definitions used?

Reply:
they are not used now, but may be in future.  I prefer to add it by one shot

Reply:
I think it makes sense to add them when used

Reply:
I'm still not in favor of adding macros that are not yet used in this patch. But if everyone else is OK with it, please go ahead with it.

Reply:
It's not so bad, and it also explains why the LL/DP definitions don't start at zero.